cli/start.go

```diff
@@ -252,3 +246,97 @@ func MakeStartCommand() *cobra.Command {
 		"Skip generating an encryption key. Encryption at rest will be disabled. WARNING: This cannot be undone.")
 	return cmd
 }
+
+func addEncryptionKey(kr keyring.Keyring, cfg *viper.Viper, opts []node.Option) ([]node.Option, error) {
```

Comment:
suggestion: `getOrCreateEncryptionKey` better reflects this function

Reply:
done